docs/adr/0043-react-functional-tests.md

```diff
@@ -0,0 +1,170 @@
+- the desired end state of the react functional test migration is not universally agreed upon
+- Multiple react routes at different phases of rollout means that conditional logic is required to navigate between react/backbone test steps
+- The tests in `react-conversion` will need to be modified again during the cleanup phase to remove all of the feature flag check and experiment parameters
+
+### Questions
+
+- Are there new pattern that would simplify conversion moving forward (e.g., user-facing locators, single page POMs, route check in POMs, etc.)
+- Do we plan to discontinue all backbone tests? If so, when should we remove vs skip?
+- For tests that are not specific to the react conversion routes but that include test steps that use the converted routes, should we create new test files, or update the POM to be react/backbone agnostic?
+- How should we handle test setup (account creation and signup in fixtures) that currently use the backbone POMs?
+- How could we modify our test architecture to support coverage of both backbone and react tests while a route group experiment is enabled but not fully rolled out?
+- Would updating the locators to be backbone/react agnostic resolve the majority of failures? (currently unknown without doing the work)
+- Can we modify top-level fixtures to enroll in the react experiment instead of including `forceExperiment` params in all test steps?
+
+## Decision Drivers
+
+1. Test Coverage
+
+The chosen option should ideally maintain or increase coverage for React flows and avoid compromising (skipping) coverage for Backbone flows.
+
+2. Time & Effort
+
+The chosen option should ideally be feasable within the planned timeframe for the signin React rollout (Train 285).
+
+3. Best Practice & Technical Debt
+
+The chosen option should incorporate lessons learned and avoid contributing to future maintenance by:
+
+- Using fixtures to setup and teardown stateful dependencies, such as accounts
+- Using built-in user-facing locators
+- Using auto-awaiting asserts
+- Avoiding conditional paths
+- Minimizing use of skip and fixme annotations
+
+## Considered Options
+
+- Option 1: Single set of POMs for React/Backbone, update existing tests
+  - Option 1a: Update by Page-First Approach
+  - Option 1b: Update by Test-First Approach
+- Option 2: Status Quo - Separate POMs for React and Backbone, Migrate Tests According to [Analysis][2]
+
+## Decision Outcome
+
+Chosen option: **Option 1b, Single set of POMs for React/Backbone, update existing tests with a test-first approach**
+
+Option 1 chosen over 2 to reduce the number of tests that need to be duplicated to support the react conversion and prioritize maintaining a full suite of tests that can support both backbone and react, while implementing best practices and reducing tech debt.
+
+Option 1b chosen over 1a because it allows for prioritizing fixing the failing tests for the react signin full prod rollout.
+
```

Comment:
In this section perhaps we can rephrase to link the chosen decision more directly with the above criteria. Something like:

```suggestion
## Decision Outcome

Chosen option: **Option 1b, Single set of POMs for React/Backbone, update existing tests with a test-first approach**

Overall, option 1b best fulfills our time & effort decision criteria, keeping the signin React rollout on track. While the action plan outlined in option 2 has served the team well thus far, separating the React and Backbone concerns, it will not scale in the signin use case, where the POM and test case duplication required would considerably extend the timeline for releasing the React signin feature set. 

All options will maintain the same amount of test coverage, however option 1 better assures that requirement changes are universally applied to test cases for React and Backbone.

Option 1a offers the most clarity and efficiency in ensuring best practices and minimizing maintenance costs while moving towards are ideal POM state, but it's lack of focus on the blockers for the React signin rollout risks causing delays due to scope creep.

```

Please feel free to change/expand/ignore